node_modules/.bin/ng

```diff
@@ -0,0 +1 @@
+../@angular/cli/bin/ng
```

Comment:
there should not be a top-level node_modules at all, since each of the 3 packages has a different set of dependencies. `ng` is only a dependency for the front-end. additionally, `node_modules` is effectively a form of build output, and should absolutely never be committed to version control.